querydsl-jpa/src/main/java/com/querydsl/jpa/HibernateHandler.java

```diff
@@ -32,7 +33,7 @@
  * @author tiwe
  *
  */
-class HibernateHandler implements QueryHandler {
+public class HibernateHandler implements QueryHandler {
```

Comment:
I must be missing something - why is this public now?

Reply:
Its not necessairy. The issue reporter mentioned not being able to work around the issueX, because HibernateHandler couldnt be extended or instantiated.  opening it up for extension allows users to use the Hibernate handler with custom Templates as well (for example Templates with custom functions). For that use case the class can easily be copied as well.

The change is unrelated; so if you dont feel like exposing this class I happily revert the change :)

Reply:
Sounds good to me.